README.md

```diff
@@ -288,6 +266,58 @@ The example output in the [data/examples](data/examples) folder shows the output
   * Restart DSX Desktop
   * Load the jupyter notebook again
 
+* If you got this error while running this cell
```

Comment:
Lets follow Nick's format. Error and Solution format. probably easier to consume.